go.sum

```diff
@@ -115,6 +116,7 @@ github.com/cosmos/cosmos-sdk v0.34.4-0.20201022224044-5bebf2bd39cd h1:5UiE2afJO8
 github.com/cosmos/cosmos-sdk v0.34.4-0.20201022224044-5bebf2bd39cd/go.mod h1:KvZzKUEc7KrMiiIeU0zKPMMwuxxn+R0Leo8vBSDgOeQ=
 github.com/cosmos/cosmos-sdk v0.34.4-0.20201028155748-1b0d6540d8d5 h1:ALPZHfvrFRMyFDhGi3OTzHo7DUN+VENVNIwAnWOqHwQ=
 github.com/cosmos/cosmos-sdk v0.34.4-0.20201028155748-1b0d6540d8d5/go.mod h1:KvZzKUEc7KrMiiIeU0zKPMMwuxxn+R0Leo8vBSDgOeQ=
+github.com/cosmos/cosmos-sdk v0.39.1 h1:vhjf9PZh9ph8btAj9aBpHoVITgVVjNBpM3x5Gl/Vwac=
```

Comment:
these go.sum / go.mod updates may not be necessary?

Reply:
Updated both files by running `go mod tidy`.